src/style.css

```
@@ -0,0 +1,168 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+}
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+/*Big form*/
+
+.big__box {
+  position: relative;
+  height: 70px;
+  margin: 30px 0;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  transition: 0.2s all;
+}
+
+.big__box:hover {
+  position: relative;
+  height: 70px;
+  margin: 30px 0;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+  transition: 0.2s all;
+}
```

```suggestion
.big__box:hover {
  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
}
```
Don't repeat yourself. You just only change box-shadow on hover